community/demos.md

```diff
@@ -7,6 +7,11 @@ markdown file in `community/demos/`.
   - Blog post about Cloud Events Orchestration using an workflow engine([Zeebe](http://zeebe.io)). 
   - Instructions: You can find the [Demo Code and instructions here](https://github.com/salaboy/zeebe-cloud-events-examples).
   
+- [Gaining business visibility into processes implement by Cloud Events](https://blog.bernd-ruecker.com/gaining-visibility-into-processes-spanning-multiple-microservices-a1fc751c4c13s)
```

Comment:
I get a 404 from this url.

Reply:
https://blog.bernd-ruecker.com/gaining-visibility-into-processes-spanning-multiple-microservices-a1fc751c4c13
works though. I think the 's' at the end of the url is messing it up

Reply:
Good catch - sorry for that - fixed!